wbooks/src/app/screens/BookList/index.tsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-use-before-define */
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+
+import BookItem from './components/BookItem';
+
+const BookList = () => (
+  <View style={styles.container}>
+    <BookItem />
+    <BookItem />
+    <BookItem />
+    <BookItem />
+  </View>
+);
+
+const styles = StyleSheet.create({
+  container: {
+    // flex: 1,
+  }
+});
```

Comment:
Split styles, also you can use here the styles that you declare in the `app/index` View